npc/re/jobs/novice/academy.txt

```diff
@@ -13,80 +13,226 @@
+		mes "At least you are save.";
+		mes "Are you alright?";
+		next;
+		if (select("Im alright, but others need help.", "I think I am the last?") == 1) {
 			mes "[Captain Carocc]";
-			mes "Let me give you more details later.";
-			mes "See you back at the harbor deck.";
-			next;
+			mes "There are more people left?";
+			mes "I will send a rescue team to them";
+			mes "Thank you for your report";
+		} else {
 			mes "[Captain Carocc]";
-			mes "To get off this ship, you should enter the ^4d4fffShining Portal^000000 over there.";
-			mes "All transportation is made through the portals.";
-			setquest 7471;
-			completequest 7471;
-			close2;
-			cutin "", 255;
-			end;
+			mes "Alright, thats good.";
+			mes "The storm came unexpected but I never thought it hit us that hard.";
 		}
+		next;
 		mes "[Captain Carocc]";
-		mes "Now, we are here in Izlude~";
-		mes "Go through that ^4d4fffShining Portal^000000 and exit outside.";
+		mes "At least, most people are already in safety.";
 		next;
 		mes "[Captain Carocc]";
-		mes "Let's talk more when we get off to the deck.";
-		mes "There are lots of things I want to teach you.";
-		close2;
+		mes "Look at the trees there, ^4d4dfffollow them to the southwest^000000 to find a new ship.";
+		mes "The Ship will take you to ^4d4dffIzlude^000000, our original destination.";
+		next;
+		mes "[Captain Carocc]";
+		mes "Ah! Speaking of it, those ^4d4dffPorings^000000 on this island drop several material we could need for reparation.";
+		next;
+		mes "[Captain Carocc]";
+		mes "^4d4dffKill the Porings and bring 2 Lumber to the southern crew^000000.";
+		next;
+		mes "[Captain Carocc]";
+		mes "It is hard, but you look tough enough for this task.";
+		if (checkquest(21001) == 1)
+			completequest(21001);
+		specialeffect2 EF_HEAL2;
+		heal 9999, 0;
+		skilleffect 34,0;
+		sc_start SC_BLESSING,240000,10;
+		skilleffect 29,0;
+		sc_start SC_INCREASEAGI,240000,10;
+		getexp 600, 0;
+		setquest 21002;
+		next;
+		mes "[Captain Carocc]";
+		mes "Speaking of Porings, you have to equip your gears.";
+		mes "You should have a^4d4dffweapon and an armor^000000 equipped.";
+		next;
+		cutin "tutorial03", 4;
+		mes "!- Information -!";
+		mes "^4d4dffYou can equip gear or use items by simply double click them.";
+		mes "Item window can be opened by pressing Alt + E or using the buttons.^000000";
+		next;
+		mes "!- Information -!";
+		mes "^4d4dffEquipped Items can be seen by pressing ALT + Q.";
+		mes "Either double click or drag & drop them.^000000";
+		next;
 		cutin "", 255;
-		end;
+		mes "[Captain Carocc]";
+		mes "Kill Porings and bring the Lumber to the crew.";
+		mes "Good Luck!";
+		close;
 	}
-}
+	else {
+		mes "[Captain Carocc]";
+		specialeffect2 EF_HEAL2;
+		heal 9999, 0;
+		skilleffect 34,0;
+		sc_start SC_BLESSING,240000,10;
+		skilleffect 29,0;
+		sc_start SC_INCREASEAGI,240000,10;
+		mes "^4d4dffThe Ship^000000 anchored in the southwest. It will take you to Izlude.";
```

Comment:
`mes "^4d4dffThe Ship^000000 is anchored in the southwest. It will take you to Izlude.";`